Dockerfile-alpine.template

```diff
@@ -1,17 +1,13 @@
 # vim:set ft=dockerfile:
 FROM alpine:%%ALPINE-VERSION%%
 
-# alpine includes "postgres" user/group in base install
-#   /etc/passwd:22:postgres:x:70:70::/var/lib/postgresql:/bin/sh
-#   /etc/group:34:postgres:x:70:
-# the home directory for the postgres user, however, is not created by default
-# see https://github.com/docker-library/postgres/issues/274
-RUN set -ex; \
-	postgresHome="$(getent passwd postgres)"; \
-	postgresHome="$(echo "$postgresHome" | cut -d: -f6)"; \
-	[ "$postgresHome" = '/var/lib/postgresql' ]; \
-	mkdir -p "$postgresHome"; \
-	chown -R postgres:postgres "$postgresHome"
+# 70 is the standard uid/gid for "postgres" in Alpine
+# https://git.alpinelinux.org/aports/tree/main/postgresql/postgresql.pre-install?h=3.11-stable
+RUN set -eux; \
+	addgroup -g 70 -S postgres; \
+	adduser -u 70 -S -D -G postgres postgres; \
```

Comment:
I think we should also set `/var/lib/postgresql` as the home directory here, like we do in Debian:

https://github.com/docker-library/postgres/blob/0d0485cb02e526f5a240b7740b46c35404aaf13f/Dockerfile-debian.template#L18

(and like Alpine has done previously and still does in https://git.alpinelinux.org/aports/tree/main/postgresql/postgresql.pre-install?h=3.11-stable)